admin/app/components/solidus_admin/ui/table/component.rb

```diff
@@ -152,6 +152,6 @@ def current_scope_name
   end
 
   def initial_mode
-    @initial_mode ||= @search.value[@search.searchbar_key] ? "search" : "scopes"
+    @initial_mode ||= @search && @search.value[@search.searchbar_key] ? "search" : "scopes"
```

Comment:
Not sure why rubocop let this through, anyway I think it can be simply `@search&.value[…] …`

Reply:
The safe navigation operator doesn’t extend to the array access part, so I avoided that approach 😞 
<img width="707" alt="Screenshot 2023-11-29 at 18 19 37" src="https://github.com/solidusio/solidus/assets/19948291/c0082036-44ce-4860-962c-e85aa73cfae5">